packages/console/src/onboarding/pages/SignInExperience/index.tsx

```diff
@@ -65,10 +65,16 @@ const SignInExperience = () => {
 
   useEffect(() => {
     if (signInExperience) {
-      reset(parser.signInExperienceToOnboardSieConfig(signInExperience));
+      const sieConfig = parser.signInExperienceToOnboardSieConfig(signInExperience);
+      reset(sieConfig);
       updateAuthenticationConfig();
+
+      // Note: disable dark mode for the onboarding live preview
+      if (sieConfig.isDarkModeEnabled) {
+        setValue('isDarkModeEnabled', false, { shouldDirty: true });
+      }
     }
-  }, [reset, signInExperience, updateAuthenticationConfig]);
+  }, [reset, setValue, signInExperience, updateAuthenticationConfig]);
```

Comment:
what's this for?

Reply:
This is to set "isDarkModeEnabled" to false here when it is set to true in the sie configuration returned by the backend. Although the default value returned by the backend is currently false, this logic is added to prevent any impact on the front end caused by backend changes.